ep/build-velox/src/get_velox.sh

```diff
@@ -16,8 +16,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=2024_12_23
+VELOX_REPO=https://github.com/WangGuangxin/velox.git
+VELOX_BRANCH=feat_json_array_length
```

Comment:
@WangGuangxin, you need create a branch based on `2024_12_23` branch of oap-project, then apply your upstream velox patch.

Reply:
Got it